components/tikv_util/src/quota_limiter.rs

```diff
@@ -221,22 +322,36 @@ impl ConfigManager for QuotaLimitConfigManager {
         change: ConfigChange,
     ) -> std::result::Result<(), Box<dyn std::error::Error>> {
         if let Some(cpu_limit) = change.get("foreground_cpu_time") {
-            self.limiter.set_cpu_time_limit(cpu_limit.into());
+            self.quota_limiter
+                .set_foreground_cpu_time_limit(cpu_limit.into());
         }
         if let Some(write_bandwidth) = change.get("foreground_write_bandwidth") {
-            self.limiter
-                .set_write_bandwidth_limit(write_bandwidth.clone().into())
+            self.quota_limiter
+                .set_foreground_write_bandwidth_limit(write_bandwidth.clone().into())
         }
         if let Some(read_bandwidth) = change.get("foreground_read_bandwidth") {
-            self.limiter
-                .set_write_bandwidth_limit(read_bandwidth.clone().into());
+            self.quota_limiter
+                .set_foreground_read_bandwidth_limit(read_bandwidth.clone().into());
         }
         if let Some(duration) = change.get("max_delay_duration") {
             let delay_dur: ReadableDuration = duration.clone().into();
-            self.limiter
+            self.quota_limiter
                 .max_delay_duration
                 .store(delay_dur.0.as_nanos() as u64, Ordering::Relaxed);
         }
+
+        if let Some(cpu_limit) = change.get("background_cpu_time") {
```

Comment:
are we supporting change `support_auto_tune` online? If not yes, you should add `#[online_config(skip)]` annotation at `src/server/config.rs` for field `support_auto_tune`.